src/components/app.jsx

```diff
@@ -1,19 +1,73 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import '../styles/app.scss';
+import SearchForm from './search-form';
 
-const App = props => (
-  <LocationDetails
-    city={props.location.city}
-    country={props.location.country}
-  />
-);
-
-App.propTypes = {
-  location: PropTypes.shape({
-    city: PropTypes.string,
-    country: PropTypes.string,
-  }).isRequired,
-};
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
+    this.handleForecastSelect = this.handleForecastSelect.bind(this);
+    this.handleCitySearch = this.handleCitySearch.bind(this);
```

Comment:
Binding the scope manually like this does work, but you can save yourself a line of code by declaring your methods as arrow functions.
```
handleForecastSelect = () => {
}
```

Reply:
This way you wont need the binding statements.